### PR TITLE
Fix for webhook not picking up the sessionID correctly

### PR DESCRIPTION
**Pull request by jamesgarton**

### CODE REVIEW

File: src/Message/CompleteRedirectPurchaseRequest.php [hunk 1]
```diff
@@ -37,11 +39,29 @@ public function sendData($data): CompleteRedirectPurchaseResponse
...
+     * @throws InvalidRequestException
+     */
     public function getEndpoint(): string
     {
-        return sprintf($this->apiResourceString, parent::getEndpoint(),
+        $sessionID = $this->httpRequest->query->get('sessionId');
+        if (!$sessionID) {
+            // If this is from a webhook, we get the full transaction details in the post body
+            $json = json_decode($this->httpRequest->getContent(), true);
+            if ($json && !empty($json["sessionId"])) {
```
Comment (hkwak) [inline src/Message/CompleteRedirectPurchaseRequest.php]: @jamesgarton There is not need to test $json when you use empty function to test $json["sessionId"]

**Review by hkwak — APPROVED** [review]
